lighthouse-extension/app/src/popup.js

```diff
@@ -37,6 +37,32 @@ document.addEventListener('DOMContentLoaded', _ => {
   const optionsList = document.body.querySelector('.options__list');
   const okButton = document.getElementById('ok');
 
+  function getLightHouseVersion() {
+    return chrome.runtime.getManifest().version;
+  }
+
+  function getChromeVersion() {
+    return /Chrome\/([0-9.]+)/.exec(navigator.userAgent)[1];
+  }
+
+  function buildReportErrorLink(err) {
+    const reportErrorEl = document.createElement('a');
+    reportErrorEl.className = 'button button--report-error';
```

Comment:
A button would be consistent with the existing HTML buttons, but would have a few small hoops to jump through, i.e. storing the link on a `data-issue-link` attribute and binding an onclick function to the button. Nothing too crazy, but a link was easiest 😄